GeeksCoreLibrary/Core/Cms/CmsComponent.cs

```diff
@@ -231,22 +233,25 @@ protected void HandleDefaultSettingsFromComponentMode()
         /// <returns>A <see cref="DataTable" /> with the result(s), or NULL if the query was empty.</returns>
         protected async Task<DataTable> RenderAndExecuteQueryAsync(string queryToUse, DataRow dataRowForReplacements = null, bool doVariablesCheck = false, bool skipCache = false)
         {
+            //return await ComponentsService.RenderAndExecuteQueryAsync(queryToUse, ExtraDataForReplacements, dataRowForReplacements, doVariablesCheck, skipCache);
```

Comment:
Please don't leave commented code. If the code is not necessary it can be removed.